browser/services/platform.js

```diff
@@ -36,13 +36,20 @@ class Platform {
   static isVirtualizationEnabled() {
     return Platform.identify({
       win32: function() {
-        // put implementation here based on child_process.spawn method
-        // run systeminfo command and analyse stdout
-        // you should look for 'Virtualization Enabled In Firmware: Yes' line
-        // in output and if value is Yes resolve promise to true or false otherwise
-        // if output has no Virtualization Enabled In Firmware return undefined
-        // do not forget tests for platform-test.js
-        return Promise.resolve(undefined);
+        return new Promise( function(resolve, reject) {
+          const isvirtualization = require('child_process').spawn('powershell.exe', ['(GWMI Win32_Processor).VirtualizationFirmwareEnabled']);
+          isvirtualization.stdout.on('data', (data) => {
+            if(data == "True") {
+              resolve(true);
+            }
+            if(data == "False") {
+              reject(false);
+            }
+            else {
+              reject(undefined);
```

Comment:
just call resolve/reject without parameters and that is the same as passing undefined.